.github/workflows/release.yaml

```
@@ -280,6 +280,12 @@ jobs:
           sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${{ needs.tag.outputs.desktopVersion }}\",#g" packages/ui/package.json
           cd packages/ui && yarn build && yarn publish --tag latest --no-git-tag-version --new-version "${{ needs.tag.outputs.desktopVersion }}" --access public
 
+      - name: Publish Podman Extension API to npmjs
+        run: |
+          echo "Using version ${{ needs.tag.outputs.desktopVersion }}"
+          sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${{ needs.tag.outputs.desktopVersion }}\",#g" extensions/podman/packages/api/package.json
+          cd extensions/podman/packages/api && yarn publish --tag latest --no-git-tag-version --new-version "${{ needs.tag.outputs.desktopVersion }}" --access public
```

To have dependency to @podman-desktop/api and @podman-extension/api 
```suggestion
          cd extensions/podman/packages/api && yarn add @podman-desktop/api@${{ needs.tag.outputs.desktopVersion }} && yarn publish --tag latest --no-git-tag-version --new-version "${{ needs.tag.outputs.desktopVersion }}" --access public
```

---

The publishing is already using `needs.tag.outputs.desktopVersion` as version.

---

next-build.yaml don't but I think it's the same issue in this job ?